homeassistant/components/wallbox/const.py

```diff
@@ -50,49 +54,57 @@ class WallboxSensorEntityDescription(SensorEntityDescription):
         precision=2,
         native_unit_of_measurement=POWER_KILO_WATT,
         device_class=DEVICE_CLASS_POWER,
+        state_class=STATE_CLASS_MEASUREMENT,
     ),
     CONF_MAX_AVAILABLE_POWER_KEY: WallboxSensorEntityDescription(
         key=CONF_MAX_AVAILABLE_POWER_KEY,
         name="Max Available Power",
         precision=0,
         native_unit_of_measurement=ELECTRIC_CURRENT_AMPERE,
         device_class=DEVICE_CLASS_CURRENT,
+        state_class=STATE_CLASS_MEASUREMENT,
     ),
     CONF_CHARGING_SPEED_KEY: WallboxSensorEntityDescription(
         key=CONF_CHARGING_SPEED_KEY,
         icon="mdi:speedometer",
         name="Charging Speed",
         precision=0,
+        state_class=STATE_CLASS_MEASUREMENT,
     ),
     CONF_ADDED_RANGE_KEY: WallboxSensorEntityDescription(
         key=CONF_ADDED_RANGE_KEY,
         icon="mdi:map-marker-distance",
         name="Added Range",
         precision=0,
         native_unit_of_measurement=LENGTH_KILOMETERS,
+        state_class=STATE_CLASS_TOTAL_INCREASING,
```

Comment:
This seems incorrect to me? Is this sensor ever growing?

Reply:
yes, this grows during charging and then resets itself. Similar to added power, but it translates power to the amount of KM's you get out of it. I use Total increasing for these sensors as I don't have a last-reset moment. (And I want added_energy to work with the energy dashboard)

Reply:
I notice on the developers site that : "It's recommended to use state class total without last_reset whenever possible", this entity can be such a total if that is more desirable. (as long as we keep the `added_energy` entity as total_increasing for use in the energy dashboard)